.github/actions/artifact_failure/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
             docker logs "$line" > .databend/docker/"$line".log
         done
 
-        find -type d -name .databend -print0 | xargs -0 tar -zcf target/failure-${{ inputs.name }}.tar.gz
+        { printf "nohup.out\0"; find -type d -name .databend -print0; } | xargs -0 tar -zcf target/failure-${{ inputs.name }}.tar.gz
```

Comment:
This would fail if `nohup.out` not exists?

Reply:
You are right...